src/renderers/common/TimestampQueryPool.js

```diff
@@ -0,0 +1,29 @@
+class TimestampQueryPool {
```

Comment:
Do you mind adding the remaining documentation for the the ctor, properties the `dispose()` method and the class itself? It would be also good to document the derived classes (at least the bits which are added to the base implementation)

Reply:
Thanks for the review! Updated the PR with your feedbacks.